configuration/cdap-env.xml

```diff
@@ -20,7 +20,7 @@
 
   <property>
     <name>apt_repo_url</name>
-    <value>http://repository.cask.co/ubuntu/precise/amd64/cdap/3.4</value>
+    <value>http://repository.cask.co/ubuntu/precise/amd64/cdap/3.6</value>
```

Comment:
The develop branch hasn't had its version bumped, yet, allowing this simple merge. That version will be bumped in its own PR.